spk/sickchill/src/sickchill.sc

```diff
@@ -0,0 +1,6 @@
+[sickchill]
+title="SickChill"
+desc="SickChill"
+port_forward="yes"
+dst.ports="8081/tcp"

```

Comment:
That file is not referred in Makefile. By the way, it can be discarded as same content is automatically generated by generic service support from declared port.

Reply:
As @ymartin59 commented, you can delete this file.